### PR TITLE
Fix/addministrative unit without address and contact

### PR DESCRIPTION
**Pull request by aliokan**
Related to OP-3251, Follow-up of https://github.com/lblod/frontend-organization-portal/pull/621

- Remove contact and address validations in new/edit administrative-unit form when edit flag is off
- Handle administrative-unit without address and contact

### CODE REVIEW

File: app/controllers/administrative-units/new.js [hunk 1]
```diff
@@ -144,13 +145,18 @@ export default class AdministrativeUnitsNewController extends Controller {
...
+    if (this.features.isEnabled('edit-contact-data')) {
+      yield Promise.all([
+        address.validate(),
+        contact.validate(),
+        secondaryContact.validate(),
+      ]);
+    }
+
     if (!this.hasValidationErrors) {
       const siteTypes = yield this.store.findAll('site-type');
```
Comment (mirdono) [inline app/controllers/administrative-units/new.js:161]: **Nitpick:** I assume this call can move into the conditional `edit-contact-data` block. (Not sure why it is called all the way up here.)

File: app/templates/administrative-units/administrative-unit/core-data/edit.hbs [hunk 2]
```diff
@@ -346,7 +346,9 @@
               Primaire contactgegevens 
             </:title>
             <:infotext>
-               De primaire contactgegevens kunnen niet langer in het OP aangepast worden. De organisaties kunnen deze gegevens zelf aanpassen via de Contactapp.
```
Comment (mirdono) [inline app/templates/administrative-units/administrative-unit/core-data/edit.hbs:349]: **Comment:** Imo the resulting page looks a bit weird for an organisation without an address. If the organisation has no address yet the data card shows an empty box underneath the message:
2024-06-03-184013_995x114_scrot

Could we make in a bit more obvious that an address has just not been added yet?

File: app/controllers/administrative-units/administrative-unit/core-data/edit.js [hunk 3]
```diff
@@ -135,60 +135,67 @@ export default class AdministrativeUnitsAdministrativeUnitCoreDataEditController
...
+      ]);
+    }
+
     if (!this.hasValidationErrors) {
-      let primarySite = yield administrativeUnit.primarySite;
-
-      // TODO : "if" not needed when the data of all administrative units will be correct
-      // they should all have a primary site on creation
-      if (!primarySite) {
-        primarySite = primarySite = this.store.createRecord('site');
```
Comment (mirdono) [inline app/controllers/administrative-units/administrative-unit/core-data/edit.js:151]: **Nitpick:** I guess one `primarySite = ` is sufficient here.

**Review by mirdono — APPROVED** [review]
Some minor comments and nitpicks, nothing blocking for me.

File: app/templates/administrative-units/administrative-unit/core-data/edit.hbs (hunk 2 above)
Comment (aliokan) [reply app/templates/administrative-units/administrative-unit/core-data/edit.hbs:349]: I agree, I will deploy and see what Sofi think about

**Pull request merged**